ai/states/__init__.py

```diff
@@ -1,4 +1,4 @@
 # Under MIT License, see LICENSE.txt
 
-from .game_state import GameState
-from .play_state import PlayState
+#from .game_state import GameState
```

Comment:
why?

Reply:
In dev, if you try to import some functions using the usual import mechanism in regular python3 at the root of the project:
> PYTHONPATH=. python3

Some imports will fail with the current relative imports if done in isolation:

> from ai.Algorithm.evaluation_module import closest_players_to_point


It will succeed, however, if the GameState has been imported once, *before* the import line discussed above:

> from ai.states.game_state import GameState

(trying to import using the first line twice in the Python interpreter will actually work.)

This is buggy behavior, trying to import a single function should succeed instead of having to manage edge cases like this one, hence the replacement of the relative import of classes into a namespace right above it.

trying to import the closest_players_to_point:
- makes the evaluation_module try to import the GameState module
- which triggers the \_\_init\_\_.py file
- which tries to import the PlayState
- which tries to import the StrategyBook
- which tries to import various strategies that require the closest_players_to_point function.

This is a circular import situation. Also, testing the closest_players_to_point function __should not__ need to import that many modules since it does not need them at all.

Reply:
Oh! That's explains the weird bug I had with nosetest. I add the hardcode the python path to fix it.

Reply:
that's sad